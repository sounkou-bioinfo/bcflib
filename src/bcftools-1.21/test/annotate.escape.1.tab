chr1	101	=_and_%_and_;_work	%_and_=_and_:_work
