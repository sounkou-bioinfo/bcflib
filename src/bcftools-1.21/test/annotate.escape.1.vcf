##fileformat=VCFv4.3
##INFO=<ID=ISTR,Number=1,Type=String,Description="string">
##FORMAT=<ID=FSTR,Number=1,Type=String,Description="string">
##contig=<ID=chr1>
##reference=ref.fa
#CHROM	POS	ID	REF	ALT	QUAL	FILTER	INFO	FORMAT	smpl
chr1	101	.	C	G	.	.	.	.	.
