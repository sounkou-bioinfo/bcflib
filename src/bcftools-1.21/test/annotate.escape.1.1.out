##fileformat=VCFv4.3
##FILTER=<ID=PASS,Description="All filters passed">
##INFO=<ID=ISTR,Number=1,Type=String,Description="string">
##FORMAT=<ID=FSTR,Number=1,Type=String,Description="string">
##contig=<ID=chr1>
##reference=ref.fa
#CHROM	POS	ID	REF	ALT	QUAL	FILTER	INFO	FORMAT	smpl
chr1	101	.	C	G	.	.	ISTR=%3D_and_%_and_%3B_work	FSTR	%_and_=_and_%3A_work
